### ISSUE TITLE
failed to open stream: No such file or directory when seeding production

### ISSUE THREAD

**Issue by marlongichie** [slampenny/SmartSeeder#9]
This package is a great idea, but I am having issues getting it to work with production files.

With the `--env=production` option, the package creates a seed file in the `smartSeeds/production` directory. However when I use the `seed:run` command, it does not find the file.

Based on what I am seeing it is looking for the file in the root directory.
```
/app/database/smartSeeds/someSeeder.php
```
I even tried running the command like so `seed:make --env=production` but it's the same result.

Anyone else experiencing this? Any suggestions?

**Comment by mayppong** [comment]
Are you using Laravel 5? I just submitted a merge request to fix this. Because Laravel move the file structure so now `/database` is from the base path, it wasn't able to find `/app/database`. 

Hopefully the request get accepted soon.

**Comment by slampenny** [comment]
Branch 5.0 supports Laravel 5 as of last night. I'm still testing it or I would have updated the docs, but it works for at least seed:make and seed:run.

**Comment by marlongichie** [comment]
I don't believe the package is working as it should. Seems like a few things to sort out.

1. Throws a `mkdir(): No such file or directory` error, so had to create the smartSeeds directory manually to get it running
2. The `seed:make` command does not work unless an environment is specified
3. The `seed:run` command returns `Nothing to migrate` unless an environment is specified, and even when specified throws the error below

```
 Illuminate\Filesystem\Filesystem::requireOnce(): Failed opening required '/  
  home/vagrant/Code/Acme/database/smartSeeds/seed_2015_06_06_131  
  657_UserSeeder.php' (include_path='/home/vagrant/Code/Acme/vendor/phpunit/php-text-template:/home/vagrant/Code/Acme/vendor/phpunit/php-timer:.:/usr/share/php:/usr/share/pear') 
```

**Comment by marlongichie** [comment]
By the way, I am running Laravel 5

**Comment by slampenny** [comment]
So you tried the 5.0 branch? That works just fine for me, however the bugs you report would have been in master. In any case, I just merged the 5.0 branch in with master, so hopefully it'll work for you.
